docs/Installation.md

```diff
@@ -30,10 +30,11 @@ On Windows, you can simply set the `PATH` by using the advanced system settings.
 
 ## Building From Source
 
-Shiori uses Go module so make sure you have version of `go >= 1.14.1` installed, then run:
+Shiori uses Go module so make sure you have version of `go >= 1.14.1` installed. In the cloned git repository run:
 
 ```
 go get -u -v github.com/go-shiori/shiori
```

Comment:
I think it's better to just switch this command to `go install github.com/go-shiori/shiori`, right? That will build and install the command from source directly without cloning the repository (which is what the old documentaton wanted to achieve I guess).

What do you think?